CHANGELOG.md

```diff
@@ -6,6 +6,23 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
+- The **queue-granules** task no longer takes a "collection" in its config.
```

Comment:
Add the ticket number to these notes please. Also this would be a breaking change, correct?

Reply:
Maybe we should put all these changes in bullets under one line item and call out what changes will need to be made to the workflow configuration to support this.